crates/router/src/core/webhooks.rs

```diff
@@ -358,6 +366,86 @@ async fn disputes_incoming_webhook_flow<W: api::OutgoingWebhookType>(
     }
 }
 
+async fn bank_transfer_webhook_flow<W: api::OutgoingWebhookType>(
+    state: AppState,
+    merchant_account: storage::MerchantAccount,
+    webhook_details: api::IncomingWebhookDetails,
+    source_verified: bool,
+) -> CustomResult<(), errors::WebhooksFlowError> {
+    let response = if source_verified {
+        let payment_attempt = get_payment_attempt_from_object_reference_id(
+            &state,
+            webhook_details.object_reference_id,
+            &merchant_account,
+        )
+        .await?;
+        let request = api::PaymentsRequest {
```

Comment:
Move logic to a separate function.

Reply:
This is just calling a function to get payment_attempt and request is formed, there is no specific logic, so we can keep this code here itself.